src/index.css

```diff
@@ -0,0 +1,8 @@
+.page-container {
+    width: 300px;
+}
+
+.card-class {
```

Comment:
Everything here is a `class`, so better naming could be just `card`.
If you don't want to override the existing `card` class from bootstrap, you could name it more accurate.
E.g. `task-card`.